extensions/wikia/CreatePage/CreatePage.i18n.php

```diff
@@ -1,9 +1,8 @@
 <?php
-/** Internationalization file for /extensions/wikia/CreatePage/CreatePage extension. */
-$messages = [];
+$messages = array();
 
-$messages['en'] = [
-	'createpage-desc' => 'Allows to create a new page using Wikia\'s WYSIWYG editor',
+$messages['en'] = array(
```

Comment:
I talked with @rybmat and he did that manually last time. I think it doesn't make sense to update it manually to new syntax every time we use crowdin download script.
Let's keep syntax provided by the tool to avoid the same problems in future.